dbschema/organization.esdl

```diff
@@ -0,0 +1,29 @@
+module default {
+  type Organization extending Resource, Project::ContextAware, Mixin::Named {
+    overloaded name {
+      constraint exclusive;
+    }
+
+    acronym: str;
+    #address: str; #TODO - this needs figured out - needed on here and Partner?
+    types: array<Organization::Type>;
```

Comment:
I _think_ I determined that this is better
```suggestion
    multi types: Organization::Type;
```

> If order is important, use an [array](https://www.edgedb.com/docs/datamodel/primitives#ref-datamodel-arrays). Otherwise, multi properties are recommended.